tools/lib-alert-tree/lib_alert_tree/kubernetes.py

```diff
@@ -3,14 +3,24 @@
 from . import models
 
 
+def container_alerts(name, severity="warning", namespace="default"):
+    """Common alerts for Containers."""
+    return [
+        models.ExistingAlert(
+            alertname, severity=severity, namespace=namespace, statefulset=name
```

Comment:
It should be container here
```suggestion
            alertname, severity=severity, namespace=namespace, container=name
```

And in order to be able to use it in `pod_alerts` you will need to support both `container` and `pod`